### DISCUSSION TITLE
Support for the Outbalanced Output

### DISCUSSION THREAD

**Discussion by DevineCZ** [mletenay/home-assistant-goodwe-inverter#197] [Q&A]
Hello,

Would it be possible to add support for Outbalanced Output settings? For automation I would like to switch between Balanced and Outbalanced Output.

You can find what Outbalanced Output is in the PV Master manual (page 17, or see below).

Thank you!
![outbalanced](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/133437366/af0d45e9-dce8-4444-b5b9-205801caf039)

**Comment by mletenay** [comment]
There is "unbalanced_output" setting exposed, so you can do it by invoking **set_parameter** service.
